lib/Nagios/Check/RabbitMQ/Options.hs

```diff
@@ -29,18 +26,14 @@ checkOptions = CheckOptions
         <> short 'e'
         <> help "Name of the exchange to check")
     <*> (minThreshold <$> optional ( option auto
-        ( long "minwarning"
-        <> short 'w'
-        <> metavar "MINIMUM_WARN" )))
+        ( long "minrate"
+        <> short 'r'
+        <> metavar "MINIMUM_RATE" )))
+    <*> (maxThreshold <$> optional ( option auto
+        ( long "maxrate"
+        <> short 'R'
+        <> metavar "MAXIMUM_RATE" )))
     <*> (minThreshold <$> optional ( option auto
-        ( long "mincritical"
+        ( long "minconn"
         <> short 'c'
-        <> metavar "MINIMUM_CRIT" )))
-    <*> (maxThreshold <$> optional ( option auto
-        ( long "maxwarning"
-        <> short 'W'
-        <> metavar "MAXIMUM_WARN" )))
-    <*> (maxThreshold <$> optional ( option auto
-        ( long "maxcritical"
-        <> short 'C'
-        <> metavar "MAXIMUM_CRIT" )))
+        <> metavar "MAXIMUM_CONNECTIONS" )))
```

Comment:
I think you mean MINIMUM_CONNECTIONS here. Could we also specify separate thresholds for incoming and outgoing connections? They have different expected values.